### PR TITLE
Add some ICEs

### PR DESCRIPTION
**Pull request by fanninpm**
Let me know if 81974 is worth keeping, based on [this comment](https://github.com/rust-lang/rust/issues/81974#issuecomment-777088084) by jonas-schievink.

### CODE REVIEW

File: ices/81199.rs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#[repr(C)]
```
Comment (JohnTitor) [inline ices/81199.rs:1]: Could you use this instead? https://github.com/rust-lang/rust/issues/81199#issuecomment-763194019

**Comment by JohnTitor** [comment]
> 
> 
> Let me know if 81974 is worth keeping, based on [this comment](https://github.com/rust-lang/rust/issues/81974#issuecomment-777088084) by jonas-schievink.

Yeah, I think we can drop it as it seems like a wontfix issue.

File: ices/81199.rs (hunk 1 above)
Comment (fanninpm) [reply ices/81199.rs]: Resolved by 4a45148

**Review by JohnTitor — APPROVED** [review]
Thanks!

**Pull request merged**